components/nifosg/nifloader.cpp

```diff
@@ -942,17 +942,11 @@ namespace NifOsg
                 return;
             }
 
-            osgParticle::ParticleProcessor::ReferenceFrame rf = (animflags & Nif::NiNode::ParticleFlag_LocalSpace)
-                    ? osgParticle::ParticleProcessor::RELATIVE_RF
-                    : osgParticle::ParticleProcessor::ABSOLUTE_RF;
+            osgParticle::ParticleSystem::ParticleScaleReferenceFrame rf = (animflags & Nif::NiNode::ParticleFlag_LocalSpace)
+                    ? osgParticle::ParticleSystem::LOCAL_COORDINATES
+                    : osgParticle::ParticleSystem::WORLD_COORDINATES;
 
-            // HACK: ParticleSystem has no setReferenceFrame method
-            if (rf == osgParticle::ParticleProcessor::ABSOLUTE_RF)
-            {
-                partsys->getOrCreateUserDataContainer()->addDescription("worldspace");
```

Comment:
Keep in mind that this decription is used in the SceneManager.

Reply:
...yes but it doesn't seems to change anything